pkgs/development/python-modules/pyinstaller/default.nix

```
@@ -34,10 +36,11 @@ buildPythonPackage rec {
   dependencies = [
     altgraph
     packaging
+    python312Packages.macholib
```

Is there a reason this has to be python312Packages specifically? If not, I think we could just import it as normal

```suggestion
    macholib
```